src/Platform/Microsoft.Testing.Platform/OutputDevice/OutputDeviceManager.cs

```diff
@@ -30,7 +30,11 @@ internal async Task<ProxyOutputDevice> BuildAsync(ServiceProvider serviceProvide
             nonServerOutputDevice = GetDefaultTerminalOutputDevice(serviceProvider);
         }
 
-        return new ProxyOutputDevice(nonServerOutputDevice, useServerModeOutputDevice ? new ServerModePerCallOutputDevice(serviceProvider) : null);
+        return new ProxyOutputDevice(
+            nonServerOutputDevice,
+            useServerModeOutputDevice
+                ? new ServerModePerCallOutputDevice(serviceProvider)
+                : null);
```

Comment:
I don't like in general `null` to avoid to check around...I would update with a `NopOutputDevice` that's doing nothing(will be inlined and elided by the jitter most proabably)

Reply:
I would cache a static instance. But not so important.